js/src/models/model.users.js

```diff
@@ -220,7 +220,19 @@ Wu.User = Wu.Class.extend({
 
 	_save : function (changes) {
 		if (app.activeProject) changes.project = app.activeProject.getUuid(); // for edit_user access, may need project...
-		Wu.save('/api/user/update', JSON.stringify(changes)); 
+		app.api.updateUser(JSON.stringify(changes), function (err, result) {
```

Comment:
actually, here you should pass object, not stringified. got an error, and anyway it will be stringified later.